maps/torch/job/torch_jobs_boh.dm

```diff
@@ -298,12 +300,18 @@
 		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/security/brig_chief/marine
 	)
 	allowed_ranks = list(
+		/datum/mil_rank/fleet/w1,
+		/datum/mil_rank/fleet/w2,
+		/datum/mil_rank/fleet/w3,
 		/datum/mil_rank/fleet/e6,
 		/datum/mil_rank/fleet/e7,
 		/datum/mil_rank/fleet/e8,
+		/datum/mil_rank/marine_corps/w1,
+		/datum/mil_rank/marine_corps/w2,
+		/datum/mil_rank/marine_corps/w3,
 		/datum/mil_rank/marine_corps/e6,
 		/datum/mil_rank/marine_corps/e7,
-		/datum/mil_rank/marine_corps/e8
+		/datum/mil_rank/marine_corps/e8,
```

Comment:
Same here, `list` ends so there actually does not need to be a comma here.

Reply:
Okay thank  you

Reply:
The comma is still there.